bucket/mousejiggler.json

```diff
@@ -1,19 +1,20 @@
 {
-    "version": "1.8.42",
+    "version": "2.0.7",
     "description": "A very simple piece of software whose sole function is to \"fake\" mouse input to Windows, and jiggle the mouse pointer back and forth.",
-    "homepage": "https://github.com/cerebrate/mousejiggler",
+    "homepage": "https://github.com/cerebrate/mousejiggler/",
```

Comment:
Why do you "fix" something, which is not broken??

![image](https://user-images.githubusercontent.com/13260377/105571766-f2f86a80-5d52-11eb-83ea-c93243a63923.png)

Reply:
Just because it works doesn't mean it's not broken. 🙂

That URL points to a (directory-like) collection of resources, not a (file-like) single resource, so it should have a trailing slash. This is, admittedly, a fairly pedantic interpretation of web standards that only affects relative URLs when parsing and most web servers are sloppy enough to let me get away with omitting the /, and yet. It's still wrong.